pages/linux/cmus.md

```diff
@@ -0,0 +1,24 @@
+# cmus
+
+> Commandline C* Music Player.
```

Comment:
I think we could just remove it. It's not really relevant to know what language a tool is written in, and here I'm afraid it just add noise.